website/docs/dbt-cli/install/overview.md

```diff
@@ -15,3 +15,7 @@ You can install dbt Core on the command line by using one of these recommended m
 ## About dbt adapters
 
 dbt works with a number of different databases, query engines, and other SQL-speaking technologies. It does this by using a dedicated _adapter_ for each. When you install dbt, you'll also want to install the specific adapter for your database. For more details, see the [list of available adapters](available-adapters).
+
+## Upgrading dbt
```

Comment:
@jasnonaz I love the idea of adding an upgrade section. I think it's worthy of being its own page even if we only have pip instructions right now.

Reply:
Actually, I this is valuable information and we should get it added.